src/subcommand.rs

```diff
@@ -567,18 +585,24 @@ impl Subcommand {
               }
             }
 
-            print!(
-              "{list_prefix}{}",
-              RecipeSignature { name, recipe }.color_display(config.color.stdout())
-            );
-
-            format_doc(
-              config,
-              name,
-              doc.as_deref(),
-              max_signature_width,
-              &signature_widths,
-            );
+            if i == 0 || config.no_inline_aliases {
+              print!(
+                "{list_prefix}{}",
+                RecipeSignature { name, recipe }.color_display(config.color.stdout())
+              );
+
+              format_doc(
+                config,
+                name,
+                doc.as_deref(),
+                aliases
+                  .get(recipe.name())
+                  .filter(|_| !config.no_inline_aliases)
```

Comment:
I think this use of filter is a bit confusing. I think all aliases should always be passed to `format_doc`, but only printed in-line if `no_inline_aliases` is false.

Reply:
Switched `format_doc` to take `&Vec<&str>` instead of `Option<&Vec<&str>>` for aliases